src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -264,6 +264,23 @@ def haveEnoughRealisations(self, realizations) -> bool:
     def __ne__(self, other):
         return not self == other
 
+    def __repr__(self):
+        return (
+            "AnalysisConfig(config_dict={"
+            f"'UPDATE_LOG_PATH'={realpath(self.get_log_path())}, "
```

Comment:
should the equal sign be a colon with a space, like for the rest of the options?

Reply:
Fixed